### PR TITLE
PUB 1290 cookies page updates

### PR DESCRIPTION
**Pull request by joshblackmoor**
### JIRA link ###

https://tools.hmcts.net/jira/browse/PUB-1290

### Change description ###

- Updated cookies page with new wording
- Added welsh version of the cookie page
- Added form to enable/disable some cookies and set up analytics and performance cookie functions to delete those cookies

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/locales/cy/cookie-policy.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
```
Comment (ChrisS1512) [inline src/main/resources/locales/cy/cookie-policy.json:1]: Not specific to this file, however alongside the below we should also ensure that the cookies are disabled when the user declines them.

I believe theres code elseware in this repo that we can plugin to do this.

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Comment by kurt-uk** [comment]
cant see where the logic is that uses the functionality of actually disabling or enabling depending on the toggles; see cookie-preferences.js

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]
Currently getting a sonarqube failure on the build. Once sorted I'll give this a test

**Comment by ChrisS1512** [comment]
Just trying this out now via the DEV pod and seems to be keeping the cookies. Would you be able to try it out to see if you get the same result?

**Review by furnivall — DISMISSED** [review]
Really great work guys - <3

**Review by ChrisS1512 — DISMISSED** [review]

**Review by ChrisS1512 — APPROVED** [review]

**Review by junaidiqbalmoj — APPROVED** [review]

**Pull request merged**